### ISSUE TITLE
Add header bar

### ISSUE THREAD

**Issue by tomzemp** [pepfar-datim/datim-mechanism-finder#25]

**Comment by tomzemp** [comment]
Ran into a lot of issues here :-(

When initially using `npm install @dhis2/d2-ui-header-bar`, I got this error:
![image](https://user-images.githubusercontent.com/18490902/60460613-8683bb00-9c44-11e9-9fbb-6dce04447a63.png)

I then tried to use the exact version that @jakub-bao used in Approvals-Dashboard (5.3.11)...but still got the same error. In the end, I had to use the same configuration on most of the react modules and header-bar from Approvals-Dashboard, however, this still gave me errors with babel-jest (I had to install the specific package version used by the react app higher in the node-modules tree so that it didn't conflict with the package used by the header bar).

I think this is still a bit TBD as to what is going on. It seems like there's still an error in the header bar configuration.
